src/components/PostIndex.jsx

```diff
@@ -33,10 +33,11 @@ const PostIndex = ({ post }) => {
         <Card.Divider />
         <TouchableOpacity
           style={styles.commentButton}
+          title='Comment'
           testID='comment-button'
           onPress={() => navigation.navigate('CommentSection', { post: post })}
         >
-          Comment
+          <Text>Comment</Text>
```

Comment:
we don't need the title if we have the <Text>

Reply:
Good point!